### DISCUSSION TITLE
Bug when the mod is enabled.

### DISCUSSION THREAD

**Discussion by confusednugget** [Starystars67/BeamNG-MP#347] [Q&A]
Whenever you enable the Beammp mod, you lose the ability to highlight subparts (check video for context), I've noticed this issue for a month now, and I thought it was an issue with my game, until I decided to experiment, so I deactivated multiplayerbeammp, and I was able to highlight subparts again.

https://github.com/BeamMP/BeamMP/assets/128351854/8f885e86-143b-48d0-be13-7c0c197becb5

**Comment by confusednugget** [comment]
Also if this is the wrong location, I apologize, I'm new to this.

**Reply by Starystars67** [reply]
Hey there, thank you for raising this. 
It seems that PR #348 resolves this.

Thank you again for raising this!
